lib/ansible/modules/network/edgeos/edgeos_config.py

```diff
@@ -137,6 +142,16 @@
   returned: always
   type: list
   sample: ['...', '...']
+invalid:
+  description: The list of configuration commands removed for being invalid
```

Comment:
This makes it seem like the commands were removed from the device, not from the commands _run_ on the device.
```suggestion
  description: The list of configuration commands not run on the device since they do not begin with ``set`` or ``delete``
```